apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss

```diff
@@ -23,11 +23,23 @@
     }
 
     .plan-content {
-        padding: 8px 8px 36px 8px;
+        padding: 8px 8px 10px 8px;
 
         .plan-title {
             min-height: 72px;
             font-weight: 400;
+
+            span {
+                overflow: hidden;
+                display: -webkit-box;
+                -webkit-box-orient: vertical;
+                -webkit-line-clamp: 2;
+            }
+
+            @media screen and (max-width: 1299px) {
+                font-size: 18px !important;
```

Comment:
why use static font ?

We have available font sizes

 --font-size-16 and --font-size-20 :

Reply:
done